node_modules/oae-core/topnavigation/bundles/ca_ES.properties

```diff
@@ -1,6 +1,4 @@
 #X-Generator: crowdin.com
-SKIP_TOP_MENU=Anar a la patgina [c]
-SKIP_TOP_MENU_KEY=c
 SKIP_MENU_TARGET=El contingut comen\u00e7a a\u00e7i
```

Comment:
In the default.properties file, we are retaining the `SKIP_TO_CONTENT` key, whilst in the other language files we are retaining the `SKIP_MENU_TARGET` key